server/views/pages/dossierRecallInformation.njk

```diff
@@ -32,9 +32,9 @@
             {% include '../partials/rescinds.njk' %}
             {% include '../partials/notes.njk' %}
             <div class="govuk-form-group govuk-!-margin-top-4" >
-                {% if recall.requiredDocsMissing.length %}
+                {% if recall.missingDocuments.required.length %}
```

Comment:
Will missingDocuments always be non-null in this scenario?

Reply:
good spot. will add some `?`